projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -759,4 +767,5 @@ export const defaultFeatureToggles: Required<FeatureTogglesInterface> = {
   navigationMenuCloseOnSameLinkClick: false,
   enableQuotePurchaseOrderNumber: false,
   enableReturnOrderReturnableQuantityConsigmentFallback: false,
+  enableWithCredentialsByDefault: true,
```

Comment:
> Add a new property, write JSDocs and **set the default value false** in the core lib there:  projects/core/src/features-config/feature-toggles/config/feature-toggles.ts  (for customers)

source: https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-AddaFeatureToggleinthecodesourcecode

We'll enable it by default only in 6 months from now. And after 12 months we'll remove it entirely. It's our default policy for the feature toggles lifecycle.